firebase-android-release-dashboard/src/components/Release/BuildArtifacts/BuildArtifacts.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {Link, Typography} from "@material-ui/core";
+import {PackageIcon} from "@primer/octicons-react";
+import useStyles from "./styles.js";
+
+/**
+ * BuildArtifacts component renders the build artifact status and provides a
+ * link if available.
+ *
+ * @param {Object} props - The component's props.
+ * @param {string} props.buildArtifactStatus - The status of the build
+ * artifacts.
+ * @param {string} props.buildArtifactConclusion - The conclusion of
+ * the build artifacts.
+ * @param {string} props.buildArtifactLink - The link to the build artifacts.
+ * @return {JSX.Element} The rendered component.
+ */
+function BuildArtifacts({
+  buildArtifactStatus,
+  buildArtifactConclusion,
+  buildArtifactLink}) {
+  const classes = useStyles();
+
+  const isArtifactAvailable =
+    buildArtifactStatus === "completed" &&
+    buildArtifactConclusion === "success";
+  const artifactClass = isArtifactAvailable ?
+    classes.enabledBuildArtifacts : classes.disabledBuildArtifacts;
+  const artifactText = isArtifactAvailable ?
+    "Build Release Artifacts Job" : "Build Release Artifacts Unavailable";
+
+  return (
+    <div className={artifactClass}>
+      {isArtifactAvailable ? (
+        <Link href={buildArtifactLink} rel="noopener noreferrer">
+          <Typography
+            color="textPrimary"
+            variant="body1"
+            className={classes.buildArtifactLink}
+          >
+            <PackageIcon
+              size={24}
+              className={classes.buildArtifactIcon}
+              aria-label="Package Icon"
+            />
+            {artifactText}
+          </Typography>
+        </Link>
+      ) : (
+        <Typography
+          color="textSecondary"
+          variant="body1"
```

Comment:
This should be body2 shouldn't it?

Reply:
I fixed typography variants, including this one in https://github.com/firebase/firebase-release-dashboard/pull/14/commits/b36341a0c03c90a018edf7965394469ac3f0fc81